auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -379,28 +396,17 @@ public String getProviderId() {
             return mProviderId;
         }
 
+        @Deprecated
```

Comment:
Normally `@Deprecated` methods continue to work.  This is a breaking change for anyone using this.

Reply:
Fixed.